src/core/renderers/DroppableGroupLayout.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * ---------------------------------------------------------------------
+ * Copyright (c) 2020 EclipseSource Munich
+ * Licensed under MIT
+ * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
+ * ---------------------------------------------------------------------
+ */
+import { GroupLayout, LayoutProps, rankWith, uiTypeIs } from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  makeStyles,
+  TextField,
+} from '@material-ui/core';
+import React from 'react';
+
+import { EditorElement } from '../../editor/components/EditorElement';
+import { EditorLayout } from '../model/uischema';
+import { DroppableLayoutContent } from './DroppableLayout';
+
+const useStyles = makeStyles((theme) => ({
+  groupLabel: {
+    padding: theme.spacing(2),
+  },
+  groupLabelInput: {
+    fontSize: theme.typography.h6.fontSize,
+  },
+}));
+
+const Group: React.FC<LayoutProps> = (props) => {
+  const { uischema } = props;
+  const groupLayout = uischema as GroupLayout & EditorLayout;
+  const classes = useStyles();
+  return (
+    <EditorElement wrappedElement={groupLayout}>
+      <Card>
+        <CardHeader
+          component={() => (
+            <TextField
+              value={groupLayout.label ?? ''}
+              placeholder='Label'
+              InputProps={{
+                classes: {
+                  input: classes.groupLabelInput,
+                },
+                readOnly: true,
+              }}
+              className={classes.groupLabel}
+            />
```

Comment:
As we don't support inline editing at the moment I would prefer a `Typography` element as the UI currently indicates that there is inline editing support while there is none.